packages/adapters/gupshup-whatsapp-v2/src/utils.ts

```
@@ -0,0 +1,121 @@
+
+  static isStylingTagIntercativeType(stylingTag: StylingTag): boolean {
+    return stylingTag === StylingTag.LIST || stylingTag === StylingTag.QUICKREPLYBTN;
+  }
+
+  static validateInteractiveStylingTag(payload: XMessagePayload): boolean {
+    if (
+      payload.stylingTag === StylingTag.LIST &&
+      payload.buttonChoices &&
+      payload.buttonChoices.choices?.length <= 10
+    ) {
+      for (const buttonChoice of payload.buttonChoices?.choices) {
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)